doc/nrf/protocols/matter/overview/data_model.rst

```diff
@@ -269,6 +272,20 @@ Sensor device types
 |                   | a diagnostic of the primary sensor and issuing a cycle of the audible        |                     |                                       |
 |                   | and visual life safety alarm indications.                                    |                     |                                       |
 +-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Air Quality       | An Air Quality Sensor is a device designed to monitor and measure various    | Certifiable         |                                       |
+| Sensor            | parameters related to the quality of ambient air in indoor or outdoor        |                     |                                       |
+|                   | environments.                                                                |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Water Freeze      | A Water Freeze Detector is capable of rating and reporting the risk of water | Certifiable         |                                       |
+| Detector          | freeze based on the external factors such as temperature, humidity, pressure,|                     |                                       |
+|                   | etc.                                                                         |                     |                                       |
```

Comment:
```suggestion
| Detector          | freeze based on the external factors such as temperature, humidity, and      |                     |                                       |
|                   | pressure.                                                                    |                     |                                       |
```

Reply:
I mean that these three parameters are only example, but there could be also other ones.

Reply:
The "such as" already covers that. Though I guess using "or" would be more accurate if not all three are always used.